src/inmanta/env.py

```diff
@@ -73,22 +89,34 @@ def _init_env(self) -> bool:
         """
         self._parent_python = sys.executable
 
-        python_name = os.path.basename(sys.executable)
-
         # check if the virtual env exists
-        if sys.platform == "win32":
-            python_bin = os.path.join(self.env_path, "Scripts", python_name)
-        else:
-            python_bin = os.path.join(self.env_path, "bin", python_name)
+        if os.path.isdir(self.env_path) and os.listdir(self.env_path):
+            # make sure the venv hosts the same python version as the running process
+            if sys.platform.startswith("linux"):
+                # On linux distribs we can check the versions match because the env's version is in the site-packages dir's path
```

Comment:
"distribs", is this a typo or a French abbreviation? Just curious, the meaning is clear.

Reply:
Oh yes, I guess that's French :D I'll change to `distributions`